### PR TITLE
SOT-185: Deploy contracts

### PR DESCRIPTION
**Pull request by hduoc2003**

### CODE REVIEW

File: examples/config-files/deploy-contracts.toml
Comment (zk-steve) [inline examples/config-files/deploy-contracts.toml:1]: file name

File: Cargo.toml [hunk 1]
```diff
@@ -4,13 +4,20 @@ version = "0.1.0"
...
 anyhow = { version = "1.0.93" }
-tokio = { version = "1.41.1" }
-aptos-sdk = { git = "https://github.com/aptos-labs/aptos-core", rev = "28abb7db64af8de81a6bc76c6b2d6600181ac578" }
-serde = { version = "1.0.215" }
+aptos = { git = "https://github.com/sota-zk-labs/aptos-core", branch = "jayce-aptos-node-v1.23.5" }
+aptos-sdk = { git = "https://github.com/sota-zk-labs/aptos-core", branch = "jayce-aptos-node-v1.23.5" }
+clap = { version = "4.5.21", features = ["derive"] }
 config = { version = "0.14.1" }
+dialoguer = "0.11.0"
+rand = "0.7.3"
```
Comment (zk-steve) [inline Cargo.toml:13]: version =

File: src/utils.rs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+use crate::deploy_config::AptosNetwork;
+
+pub const DEFAULT_FAUCET_AMOUNT: u64 = 100_000_000;
+
+pub fn check_aptos_installed() -> anyhow::Result<()> {
+    ensure!(
+        Command::new("aptos").output().is_ok(),
+        "Aptos CLI not found. Please install it from https://aptos.dev/en/build/cli"
+    );
```
Comment (zk-steve) [inline src/utils.rs]: remove

**Review by zk-steve — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (hduoc2003) [reply Cargo.toml:13]: the latest version of `rand` is not compatiable

**Pull request merged**